Index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
- [ ] **OPTIONAL** It'd be way too nicer if you change the default `title` tag to something related to the project, and not just the default Document.
Keep in mind that the title of a page is one of the first things that a viewer see.